### ISSUE TITLE
AVT 1 - Tooltip DAP violation 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2458]
This issue is related to closed issue #1342 
## Environment
macOS Mojave version 10.14.4
Chrome Version 74.0.3729.131 (Official Build) (64-bit)

## Detailed Description
Run DAP with Feb. 2019 ruleset on the Tooltip Component
Violation: The WAI-ARIA widget with img role must have a nonempty alternate text or a label.
(See screenshot)
<img width="1101" alt="Screen Shot 2019-05-08 at 8 50 22 AM" src="https://user-images.githubusercontent.com/21676914/57380345-75e82480-716e-11e9-97ba-103c48304f02.png">

To fix the issue remove aria-labelledby="_carbon-tooltip-trigger_qonq7h6yqf" (see screenshot below).
<img width="979" alt="Screen Shot 2019-05-08 at 8 42 45 AM" src="https://user-images.githubusercontent.com/21676914/57380529-c19ace00-716e-11e9-9901-64c96124b446.png">

**Comment by emyarod** [comment]
alternatively, would https://github.com/carbon-design-system/carbon/pull/2656 as it is now resolve this issue?

**Comment by snidersd** [comment]
@emyarod The [DAP is available in the Chrome Web Store](https://chrome.google.com/webstore/detail/dynamic-assessment-plugin/fnapgcgphlfhecijolobjodbbnjjpdga) It includes instructions on how to select a rule set. Additional information is also on [ibm.com/able](https://www.ibm.com/able/dynamic-assessment-plug-in.html).  Please add an A11y label to this issue. Thx!

**Comment by emyarod** [comment]
@snidersd I already have DAP installed but I believe my ruleset is not up to date. Can you point me to where I can find the latest ruleset?

**Comment by snidersd** [comment]
@emyarod You select the DAP icon in the upper right corner of the browser and then select options. (See screenshots) You can find additional instructions at the links provided previously. 
<img width="1001" alt="Screen Shot 2019-05-13 at 3 38 44 PM" src="https://user-images.githubusercontent.com/21676914/57652547-6f561480-7595-11e9-86de-5c29870dba79.png">
Options will pull up a screen where you can select rulesets. 
<img width="582" alt="Screen Shot 2019-05-13 at 3 39 14 PM" src="https://user-images.githubusercontent.com/21676914/57652551-71b86e80-7595-11e9-8402-cf3556ae317e.png">

**Comment by emyarod** [comment]
@snidersd what I meant is the rulesets that are available to me seem outdated and the marketing page you provided doesn't appear to expose the latest rulesets. Could you point me to the location where I could manually apply the February 2019 ruleset?

![image](https://user-images.githubusercontent.com/8265238/57655705-19d13600-759c-11e9-95d2-87bd892e3699.png)

**Comment by dakahn** [comment]
@emyarod We have a DAP installation guide in our docs here:
https://github.com/carbon-design-system/carbon/blob/master/packages/components/docs/accessibility.md

It looks like the install link has been moved 🤦🏽‍♂️, but if you click the link for Tools you should find your way there.

@snidersd as far as I know we don't expose the _latest_ rulesets externally. I've run into the issue where the user has installed the _public_ DAP and so can't access February's ruleset 🤷🏽‍♂️

**Comment by emyarod** [comment]
@dakahn yeah I've had DAP installed for a while now, just looking for the updated ruleset

**Comment by snidersd** [comment]
@emyarod @dakahn We are checking into when the latest ruleset can be made available externally. In the meantime, I'd be happy to retest fixes if that would help?

**Comment by emyarod** [comment]
@snidersd sounds good, if you could help test the proposed fix for this issue that would be great https://github.com/carbon-design-system/carbon/pull/2656
